Tests/SwiftParserTest/Assertions.swift

```diff
@@ -198,6 +200,7 @@ func assertLexemes(
     }
     assertTokens(lexemes, expectedLexemes, markerLocations: markerLocations, file: file, line: line)
   }
+  lookaheadTracker.deallocate()
```

Comment:
Just a bit of Swift style: You can add this as a `defer`-block right after `allocate`. That way it’s even easier to verify that all allocated memory gets deallocated as well.

Same in `LexerTests.swift`